packages/entity-browser/src/routes/detail/components/ReduxFormFieldAdapter/ReduxFormFieldAdapter.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import _omit from 'lodash/omit'
+import FormField from '../../../../components/FormField'
+
+const ReduxFormFieldAdapter = props => {
+  const {
+    input,
+    meta: {dirty, touched, error, submitting},
+    id,
+    formDefinitionField,
+    entityField,
+    modelField,
+    editableValueUtils
+  } = props
+
```

Comment:
not necessary to create a new function on each render. can be moved out of the function and receive `input` as parameter.